api-schema/src/main/java/org/stellar/anchor/api/platform/PlatformTransactionData.java

```diff
@@ -57,8 +57,11 @@ public class PlatformTransactionData {
   @SerializedName("stellar_transactions")
   List<StellarTransaction> stellarTransactions;
 
-  @SerializedName("stellar_transaction_id")
-  String stellarTransactionId;
+  @SerializedName("source_account")
```

Comment:
Can we use `from_account` which is the same as the database?

Reply:
`source_account` is always user account.
So it's `from` for deposit and `to` for withdrawal

Reply:
`source_account` is always the source of the payment. In deposits, this is the anchor's account. In withdraw's, its the user's account.

Reply:
Ok, I will just mirror to and from account